### DISCUSSION TITLE
Interact with Cypress while waiting

### DISCUSSION THREAD

**Discussion by mamoedo** [cypress-io/cypress#22967] [Questions and Help]
Hello,

I wanted to ask if it's possible to interact with the web browser while the tests are running.

I have this question because I need to perform some actions on my web that I think that I can't do with Cypress so I thought about making a wait on the test and performing manually this actions.

What I can't do with Cypress is to upload files to my web because it's made with Webix and the html elements for uploading files are not input fields (and Cypress can only upload files to this kind of elements). 

Does anyone know a way to perform this actions?
Thank you very much

**Comment by stavares843** [comment]
hello! 👋 

have you tried the [cypress-file-upload](https://www.npmjs.com/package/cypress-file-upload) to upload the files? 


more details on how to use it [here](https://stackoverflow.com/a/62204487)

**Reply by mamoedo** [reply]
Yes, I've tried but it does not seem to work with this kind of element (because it requires an html input and my element is not an input but an element with a click listener)
